server/server.go

```diff
@@ -273,17 +277,14 @@ func (svr *Server) WriteResponse(w http.ResponseWriter, data []byte) {
 }
 
 func (svr *Server) WriteInternalError(w http.ResponseWriter, err error) {
-	svr.log.Error("internal server error", "err", err)
```

Comment:
Removing these because of duplicate logs (also printed in middleware). However this means we lose the error/info + status message.......

Reply:
Thinking switching to echo webserver as in https://github.com/Layr-Labs/eigenda-proxy/pull/142 would be a really good idea at this point.... @epociask let's discuss tomorrow